### DISCUSSION TITLE
Failed to build CMS_S_5

### DISCUSSION THREAD

**Discussion by prabhatsharmaa** [ARM-software/CMSIS_5#1552] [Q&A] [answered]
I am trying to build CMSIS_5 , I see missing library errors. 

**Error :** 
_File/Path not found: '/home/adp/Documents/prabhat/cmsis/CMSIS_5/./build/CMSIS/RTOS/RTX/LIB/ARM/RTX_CM0.lib'. Check_ correct spelling

-> I see that libraries aren't present int ./build/CMSIS/RTOS/RTX/LIB/ARM/ folder.
-> I also see a script to fetch these library "fetch_libs.sh". When I run this script i see following error:

**Error :**
_Please set your Artifactory in ARTIFACTORY_API_KEY

1. Browse to https://artifactory.eu02.arm.com:443/ui/admin/artifactory/user_profile
2. Copy the API Key
3. Add 'export ARTIFACTORY_API_KEY="<API Key>"' to ~/.bashrc_

However, when I browse to this url, it say URL doesn't exist.

Can anyone help me to resolve this issue?

**Comment by VladimirUmek** [comment, marked as answer]
Hi,
please see issue #1518.  If you don't need any particular fix I suggest that you simply use the [latest released version](https://github.com/ARM-software/CMSIS_5/releases).

**Reply by prabhatsharmaa** [reply]
Thank you very much! I was able to build by downloading the libraries from the released version.
